lib/msal-common/src/utils/Constants.ts

```diff
@@ -64,6 +64,14 @@ export const Constants = {
     INVALID_INSTANCE: "invalid_instance",
 };
 
+export const ErrorCodeBounds = {
+    clientErrorLowerLimit: 400,
```

Comment:
1. Naming of constants seems to be inconsistent. I see that constants are all UPPER_CASE 
2. Consider renaming the class to "HttpStatus" and the constants to `CLIENT_ERROR_LOWER_LIMIT` etc.